src/net.cpp

```diff
@@ -2247,7 +2247,9 @@ void CConnman::DisconnectNodes()
                         .grant = std::move(pnode->grantOutbound),
                         .destination = pnode->m_dest,
                         .conn_type = pnode->m_conn_type,
-                        .use_v2transport = false});
+                        .use_v2transport = false,
+                        .masternode_connection = pnode->m_masternode_connection,
+                        .masternode_probe_connection = pnode->m_masternode_probe_connection});
```

Comment:
why do we need to add `masternode_connection` and `masternode_probe_connection`, but not `pnode->fSendDSQueue`, `pnode->qwatch`, and `pnode->m_masternode_iqr_connection` ?